config/opensearch.yml.example

```diff
@@ -38,6 +38,9 @@ plugins.security.authcz.admin_dn:
 # BOTH - backend roles are mapped to Security roles mapped directly and via roles_mapping.yml in addition
 plugins.security.roles_mapping_resolution: MAPPING_ONLY
 
+# Define behavior for following referalls from LDAP reponses (default: true)
+plugins.security.FOLLOW_REFERRALS: true
```

Comment:
I don't think this should be such a high level setting, and it isn't following the standard casing convensions, I think this should be specific for an LDAP configuration, please move the setting into the LDAP specific section of the configuration

Reply:
Oops, you are correct. This was not where this should go.

Reply:
Maybe this wasn't updated?